scripts/stetho_open.py

```diff
@@ -78,9 +78,9 @@ def _find_only_stetho_socket(device):
   finally:
     adb.sock.close()
 
-def _connect_to_device(device=None):
+def _connect_to_device(device=None, port=5037):
```

Comment:
yeah, the convention here is that _ functions are "private" so this would /have/ to be specified